app-misc/imgcat/imgcat-1.1.1.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Print images on to your terminal"
+DESCRIPTION="imgcat prints images to your terminal screen. Yes, indeed it does. \
+By default, the image is printed at the full width and colour depth \
+detected for your terminal. These can be overridden using -w to adjust \
+the maximum width or -R to prevent resizing, even if the image is too \
+big to fit in the terminal; and -d to explicitly the set color depth. \
```

Comment:
Remove the last "\" for a line-break. Maybe even add a newline to make the text less crammed.